new_pages/basics.qmd

```diff
@@ -483,7 +480,7 @@ In summary: to use the functions available in an R package, 2 steps must be impl
 
 #### Your library {.unnumbered}
 
-Your "library" is actually a folder on your computer, containing a folder for each package that has been installed. Find out where R is installed in your computer, and look for a folder called "win-library". For example: `R\win-library\4.0` (the 4.0 is the R version - you'll have a different library for each R version you've downloaded).
+Your "library" is actually a folder on your computer, containing a folder for each package that has been installed. Find out where R is installed in your computer, and look for a folder called "win-library". For example: `R\win-library\4.4.1` (the 4.4.1 is the R version - you'll have a different library for each R version you've downloaded).
```

Comment:
@arranhamlet can you check if "win-library" is still relevant and change this if necessary? In my R install now, the folder to the library is "R/R-4.4.1/library"